.github/workflows/mirror-builtin-actors.yml

```diff
@@ -0,0 +1,61 @@
+name: Mirror Builtin Actors Releases
+on:
+  schedule:
+    - cron: '0 * * * *'  # Runs every hour
+  pull_request:
+    paths:
+      - 'scripts/mirror-builtin-actors.sh'
+  push:
+    paths:
+      - 'scripts/mirror-builtin-actors.sh'
+  workflow_dispatch:
+
+jobs:
+  mirror-releases-do:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up S3cmd cli tool for DigitalOcean
+        uses: s3-actions/s3cmd@v1.5.0
+        with:
+          provider: digitalocean
+          region: fra1
+          access_key: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          secret_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+      - name: Run mirroring script to DigitalOcean
+        working-directory: scripts
+        if: github.ref == 'refs/heads/main' && (github.event_name == 'push' || github.event_name == 'workflow_dispatch')
+        run: bash ./mirror-builtin-actors.sh
+        env:
+            SLACK_API_TOKEN: ${{ secrets.SLACK_TOKEN }}
+            SLACK_CHANNEL: "#forest-dump"
+            ENVIROMENT: digitalocean
+            BUCKET_NAME: filecoin-builtin-actors
+
+  mirror-releases-cf:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v2
+
+      - name: Set up S3cmd cli tool for Cloudflare
+        uses: s3-actions/s3cmd@v1.5.0
+        with:
+          provider: cloudflare
+          region: auto
+          access_key: ${{ secrets.R2_ACCESS_KEY }}
+          secret_key: ${{ secrets.R2_SECRET_KEY }}
+          account_id: "2238a825c5aca59233eab1f221f7aefb"
```

Comment:
While not extremely sensitive, the ID should not be also hidden behind a secret.

Reply:
ping @LesnyRumcajs

Reply:
I forgot how to use English. The variable should not be exposed. It should be either a secret or a variable. Being paranoid, I'd opt for it being a secret so it is hidden in the logs.

Reply:
Cloudflare account id is available under `CLOUDFLARE_ACCOUNT_ID` secret, see https://github.com/ChainSafe/forest-iac/blob/terragrunt-iac/.github/workflows/deploy-cf-prune-latest.yml#L27